### PR TITLE
Update joda-time to 2.8.2

### PR DESCRIPTION
**Pull request by dadoonet**
## Changes in 2.8.2
- DateTimeZone data updated to version 2015f
## Changes in 2.8.1
- Fixed to handle JDK 8u60 [288, 291]
  Without this fix, formatting a time-zone will print "+00:00" instead of "GMT" for the GMT time-zone
- DateTimeZone data updated to version 2015e

Note that this change is required to make #12859 working. Otherwise, JODA version packaged with elasticsearch is still 2.8.0 so the AWS plugin won't be able to work as expected with change #12859.

### CODE REVIEW

**Comment by dadoonet** [comment]
Tests passed.

File: core/src/main/java/org/joda/time/base/BaseDateTime.java [hunk 1]
```diff
@@ -73,10 +65,10 @@ public BaseDateTime() {
     /**
      * Constructs an instance set to the current system millisecond time
      * using <code>ISOChronology</code> in the specified time zone.
-     * <p/>
+     * <p>
      * If the specified time zone is null, the default zone is used.
      *
-     * @param zone the time zone, null means default zone
+     * @param zone  the time zone, null means default zone
```
Comment (bleskes) [inline core/src/main/java/org/joda/time/base/BaseDateTime.java]: little space :)

File: core/src/main/java/org/joda/time/base/BaseDateTime.java [hunk 2]
```diff
@@ -13,7 +13,6 @@
  *  See the License for the specific language governing permissions and
  *  limitations under the License.
  */
-
```
Comment (bleskes) [inline core/src/main/java/org/joda/time/base/BaseDateTime.java]: The changes to this file are fromatting only, right? If so, can you revert?

**Comment by bleskes** [comment]
LGTM.

**Pull request merged**